package.json

```diff
@@ -48,8 +44,7 @@
     "babel-loader": "^9.1.3",
     "babel-plugin-module-resolver": "^5.0.2",
     "bootstrap": "^5.0.1",
-    "chai": "^5.1.1",
-    "chai-spies": "^1.1.0",
+    "clean-webpack-plugin": "^4.0.0-alpha.0",
```

Comment:
Webpack5 does not need this plugin.
Ref: https://github.com/summernote/summernote/pull/4651/commits/eaa1944365886f1357ff45f9d73bc3cfac9282bd